code/src/Components/FourthQ.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+const FourthQuestion = ({musicType, onCheckboxChange, nextSection, backSection}) => {
+
+return (
+
+    <article>
+        <div className="text-group">
+            <h2>What type of music do you generally listen to?</h2>
+
+                <div className="checkbox-options">
+                    <label htmlFor="thegoodkind" 
+                    key="thegoodkind">
+                        <input className="checkboxes"
```

Comment:
If you add id="thegoodkind" to this input you will be able to press the text to check the checkbox 💯

Reply:
Thank you for the tip! Will do it 👍